src/shared-components/avatar/style.ts

```diff
@@ -10,21 +9,21 @@ const SIZES = {
 };
 
 const determineSize = ({
-  size = 'small',
+  avatarSize,
 }: {
-  size?: 'small' | 'medium' | 'large';
+  avatarSize: 'small' | 'medium' | 'large';
 }) => `
-  height: ${SIZES[size]}px;
-  width: ${SIZES[size]}px;
+  height: ${SIZES[avatarSize]}px;
+  width: ${SIZES[avatarSize]}px;
 `;
 
-export const Avatar = styled.img`
+const AvatarImage = styled.img`
   ${determineSize}
   background-color: ${COLORS.secondary};
   border-radius: 50%;
   overflow: hidden;
 `;
 
-Avatar.propTypes = {
-  size: PropTypes.oneOf(['small', 'medium', 'large']),
+export default {
```

Comment:
While making changes, decided to conform to the un-named default export convention we're currently exploring.

Reply:
Is the un-named default export intended to more closely match the fact a default export allows the importing code site to use whichever name for the imported entity as the author wishes?

In other words if I can rename the entity during import why is a name specified in the export itself?

Reply:
Do you have any links or info about "un-named default export" that I could read?

Reply:
@nic-git https://www.notion.so/curology/styles-ts-5d6c113fa66240829be7d9e7f8eacea0. You can also see the Slack room #mtg-eng-demos-styles-followup for the follow-up discussion.